pyrefly/lib/commands/check.rs

```diff
@@ -577,6 +586,26 @@ impl Args {
             self.config_override.site_package_path.as_deref(),
         )?;
         validate_arg("--search-path", self.config_override.search_path.as_deref())?;
+        let conflicts: Vec<_> = self
+            .config_override
+            .enable_error
+            .iter()
+            .collect::<HashSet<_>>()
+            .intersection(
+                &self
+                    .config_override
+                    .disable_error
+                    .iter()
+                    .collect::<HashSet<_>>(),
+            )
+            .copied()
```

Comment:
can save a copy here if you make separate variables for disabled_errors and enabled_errors and then have conflicts be just the intersection operation.